vowpalwabbit/cb_explore_adf.cc

```diff
@@ -171,7 +177,8 @@ void predict_or_learn_bag(cb_explore_adf& data, base_learner& base, v_array<exam
   data.action_probs.erase();
   for (uint32_t i = 0; i < num_actions; i++)
     data.action_probs.push_back({ i,0. });
-  float prob = 1.f / (float)data.bag_size;
+  vector<uint32_t> top_actions(num_actions);
```

Comment:
declare in cb_explore_adf instead?

Reply:
same question as above

Reply:
I just realized that we'll have allocation regardless due to the pdf returned. should I switch to the C-style interface to avoid allocation?